Tamil/Thirukural.md

```diff
@@ -0,0 +1,4 @@
+Friendship:
+முகநக நட்பது நட்பன்று நெஞ்சத்து
+அகநக நட்பது நட்பு.
+Meaning: The love that dwells (merely) in the smiles of the face is not friendship; (but) that which dwells deep in the smiles of the heart is true friendship.

```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.